src/components/event/EventDetails.js

```diff
@@ -522,7 +512,6 @@ class EventDetails extends Component {
 			return (
 				<SafeAreaView style = { page }>
 					<NavBar title = { viewName } back onBack = { () => Actions.pop() } />
-					{ this.renderPickMembers() }
 					<View style = { container }>
 						<View style = { iconContainer }>
 							<Ionicons style = { [icon, textColor] } name = "md-calendar" size = { iconSize } color = '#000' />
```

Comment:
Split icons (longer than 120 char) in multiple lines and add double quotes to color(s).